packages/typia-validator/package.json

```
@@ -43,13 +43,13 @@
   "homepage": "https://github.com/honojs/middleware",
   "peerDependencies": {
     "hono": ">=3.9.0",
-    "typia": "^7.0.0"
+    "typia": "^8.0.0"
```

If there are no code changes required for `@hono/typia-validator` then it should be safe to expand the range of the peer dependency instead..

```suggestion
    "typia": "^7.0.0 || ^8.0.0"
```